app/assets/stylesheets/custom.scss

```diff
@@ -56,13 +56,13 @@ body {
 
 // HEADER STUFF
 .dad-header {
-  height: 60vh;
+  height: 70vh;
   text-align: center;
 }
 
-.dad-header h1 {
-  color: #ffffff;
-  font-size: 5.2em;
+.dad-header h2, h1 {
+  color: #337ab7;
+  font-size: 4.2em;
```

Comment:
Properties should be ordered color, font-family, font-size, font-weight, position, top